gradle/scripts/lib/java-publish.gradle

```diff
@@ -8,19 +8,6 @@ configure(projectsWithFlags('publish', 'java')) {
                 suppressPomMetadataWarningsFor('optionalApiElements')
                 suppressPomMetadataWarningsFor('optionalRuntimeElements')
 
-                // Resolved versions are used to publish platform dependencies in pom XML.
-                versionMapping {
-                    usage(Usage.JAVA_RUNTIME) {
-                        fromResolutionResult()
-                    }
-                    usage(Usage.JAVA_API) {
-                        // compileOnly dependencies should not be included in the compile scope of POM. If we
-                        // don't exclude them, compileOnly dependencies are also used to resolve the final
-                        // dependency version.
-                        fromResolutionOf("runtimeClasspath")
```

Comment:
I see that we don't exclude compileOnly dependencies explicitly anymore. Are they now excluded implicitly?

Reply:
I see. Then no problem! :smile: